app/controllers/admin/topics_controller.rb

```diff
@@ -61,9 +61,6 @@ def destroy
       @topic.destroy
       flash[:success] = 'Topic deleted'
       redirect_to admin_topics_path
-    else 
-      flash[:danger] = 'You can not delete topic with questions!'
-      redirect_to admin_topics_path  
     end
```

Comment:
Its true, but if someone use something like RESTclient. I think it will be reliable not to remove this code. But i understand your logic)